### DISCUSSION TITLE
Music font requests (idea)

### DISCUSSION THREAD

**Discussion by NoahEmmert** [musescore/MuseScore#25607] [Ideas]
Hello I have some ideas if its possible to add these fonts into MuseScore
I've been looking around for realistic fonts, and I think I have some picked here. I am not sure if it is possible to adopt them to MuseScore? 
Otherwise, I have been obsessed with music fonts for awhile, and I think these are the top 4 best realistic fonts I can find. 

* RealScore https://norfonts.ma/en/product/realscore-smufl-font/ 
* The handwritten https://norfonts.ma/en/product/the-handwritten-smufl-set-for-dorico-finale-v27/ 
* NorPen https://norfonts.ma/en/product/norpen-fonts-set-for-sibelius/ 
* The Copyist https://norfonts.ma/en/product/the-copyist-smufl-font/

**Comment by ghost** [comment]
We can't include fonts which aren't open source with the software itself, but we do intend to make it possible for users to use any SMuFL fonts they have on their system, and not be limited to those which are bundled with MuseScore Studio; see https://github.com/musescore/MuseScore/issues/24761
